<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>297</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="kplabel">
           <property name="text">
            <string>Kp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="kilabel">
           <property name="text">
            <string>Ki</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="kdlabel">
           <property name="text">
            <string>Kd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="setPoint">
           <property name="text">
            <string>Set Point</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="kpEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="kdEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="kiEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="setPointEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="turningButton">
         <property name="text">
          <string>Turning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>191</width>
      <height>177</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>OPEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>CLOSE</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="COM">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>COM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="COMComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="ClearRButton">
         <property name="text">
          <string>Clear Recieve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearTButton">
         <property name="text">
          <string>Clear Transmit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>221</width>
      <height>186</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="Transmit">
       <property name="text">
        <string>Transmit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textTransmit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>258</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="Recieve">
       <property name="text">
        <string>Receive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textRecieve"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
